source/includes/_one-time-callback-docver.html.md.erb

```diff
@@ -143,7 +143,7 @@ The payload of the request can contain the following fields:
         This field contains the profile information on the individual the
         document verification should be run for. See the
```

Comment:
Change to "This field contains the profile information for the individual on which the Document verification check is run."